### DISCUSSION TITLE
Adding a new podcast

### DISCUSSION THREAD

**Discussion by smokinjo** [advplyr/audiobookshelf#3653] [Q&A]
Hello

I am looking at installing ABS, and I have one question about its usage:

It says that you can add a podcast by searching.  Does ABS have a list of lots of podcasts to choose from?  It has a database somewhere?  

Thanks for letting me know how this aspect of ABS works.

I know that there are ways to manually add a podcast.  It is the add feature that I want to understand.

Joseph

**Comment by nichwall** [comment]
ABS doesn't provide any podcasts. The search feature searches public RSS feeds from iTunes. If you have a specific RSS feed that is not listed on iTunes you can enter the RSS URL manually and search/add it that way.

**Comment by smokinjo** [comment]
Thanks for the explaination.  It at least has a search feature with a source of podcasts.  I was not thinking that ABS hostd anything, it was more the search feature us implemented.  I am happy for that.

Thanks

Joseph

**Reply by smokinjo** [reply]
I thought of another question.

YOu said that it does not host anything.

Does ABS have the ability to store the audio files for the podcasts?  Or, is it just a software that connects the listeners with the audiosource?

The audio books part fo ABS seems to store the files, so I presume that it does it for podcasts as well.

Thanks

Joseph

**Reply by nichwall** [reply]
Yes, you still store the media files on your server just like with books. I meant that ABS itself does not provide any media files, you will need to download them from an RSS feed or add them to the file system.

**Reply by smokinjo** [reply]
Thanks for confirming that the files are stored on ABS location.

Can the download be automated or done manually?

Thanks

**Reply by advplyr** [reply]
They can be automated. Your best bet is to install the software and look around. Also check the user guides on the website.
